src/VAC/MultiView.cpp

```diff
@@ -150,6 +150,12 @@ void MultiView::setActiveView(View * view)
     }
 }
 
+void MultiView::showSettingsWidget(bool isShow)
+{
+    if(activeView_ != nullptr)
+        activeView_->viewSettingsWidget()->setVisible(isShow);
```

Comment:
I think our style is to use curly braces even on single-line if

Reply:
Fixed!